community.html

```diff
@@ -37,17 +37,17 @@ <h2 property="schema:name">Where to collaborate</h2>
                 <li>
                   <p><strong>Solid Project Forum</strong></p>
                   <p>Create and respond to comments, questions, and discussions happening in the community.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Solid Forum</a>
+                  <a href="https://forum.solidproject.org" rel="rdfs:seeAlso">Go to Solid Forum</a>
                 </li>
                 <li>
                   <p><strong>Solid Matrix Chat</strong></p>
                   <p>The Solid Matrix Chat is a live chat where community members can connect.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Matrix</a>
+                  <a href="https://matrix.to/#/#solid:converser.eu" rel="rdfs:seeAlso">Go to Matrix</a>
                 </li>
                 <li>
                   <p><strong>Solid Community Group</strong></p>
                   <p>Learn and work on Solid specifications.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Community Group</a>
+                  <a href="https://matrix.to/#/#solid_specification:gitter.im" rel="rdfs:seeAlso">Go to Community Group</a>
```

Comment:
Should this link to the Matrix, or the Community Group webpage: https://www.w3.org/groups/cg/solid/ ?

Reply:
I think they should link to https://www.w3.org/groups/cg/solid/ as well (or perhaps https://www.w3.org/community/solid/, but I think the former URL is better)